intel_extension_for_transformers/llm/runtime/graph/models/model_utils/model_files.h

```diff
@@ -211,9 +211,20 @@ struct model_file_loader {
     file.read_raw(&hparams.alibi_bias_max, sizeof(float));
     file.read_raw(&hparams.clip_qkv, sizeof(float));
     hparams.par_res = file.read_u32();
+
     hparams.word_embed_proj_dim = file.read_u32();
     hparams.do_layer_norm_before = bool(file.read_u32());
+    
+    // For ChatGLM-1 & 2
+    hparams.bos_token_id = file.read_u32();
```

Comment:
As we discussed before, I think the special tokens like `eos_id`, and `pad_id` should be members of `vocab`. line 230 for reference.  And `vocab` already has `eos_id` and `bos_id`

Reply:
I will modify them in the PR of the chatglm-1 enabling.